.github/workflows/golangci-lint.yaml

```diff
@@ -18,5 +18,5 @@ jobs:
       - name: golangci-lint
         uses: golangci/golangci-lint-action@v3
         with:
-          version: v1.46.2
+          version: v1.50.0
```

Comment:
Why are we using v1.50.0?
latest version available is v1.51.2(latest)

Reply:
Done